frontend/src/metabase/plugins/index.ts

```diff
@@ -2,6 +2,7 @@ import type { ComponentType, HTMLAttributes, ReactNode } from "react";
 import { t } from "ttag";
 import type { AnySchema } from "yup";
 
+import noResultsSource from "assets/img/no_results.svg";
```

Comment:
Fix the workaround from this commit https://github.com/metabase/metabase/pull/40139/commits/114d47bb01d488f32434a19ad56be2503394473d